README.md

```diff
@@ -1,102 +1,95 @@
-# lint-prepush    
+# lint-prepush
+
 [![npm version](https://badge.fury.io/js/lint-prepush.svg)](https://www.npmjs.com/package/lint-prepush)
 [![npm downloads](https://img.shields.io/npm/dt/lint-prepush.svg)](https://www.npmtrends.com/lint-prepush)
 [![GitHub license](https://img.shields.io/github/license/theenadayalank/lint-prepush.svg)](https://github.com/theenadayalank/lint-prepush/blob/master/LICENSE)
 [![Build Status](https://travis-ci.org/theenadayalank/lint-prepush.svg?branch=master)](https://travis-ci.org/theenadayalank/lint-prepush)
 
+> Run linters on committed files of a GIT Branch
 
-> Run linters on committed files of a GIT Branch🔬
```

Comment:
The emoji use seemed like it was starting a theme, but then was applied inconsistently to the rest of the document.

If you have ideas for what emoji should accompany each section, I'd be more than happy to add them in.